VSRAD.Syntax/FunctionList/FunctionListProvider.cs

```diff
@@ -116,26 +130,30 @@ private void ActiveDocumentChanged(IDocument activeDocument)
         private void ClearFunctionList()
         {
             _functionListControl.ClearList();
```

Comment:
`_functionListControl` can be null at this point if the function list hasn't been open yet. When closing a document, we'll arrive here from `DocumentDisposed` and may hit a `NullReferenceException`. I suggest adding a null check.

Reply:
Thank you, fixed in 80f866a